src/components/TeacherComponents/helpers/lessonHelpers/lesson.helpers.ts

```diff
@@ -52,6 +52,10 @@ export const getCommonPathway = (
           acc[key] = null;
```

Comment:
> /**
 * Returns the intersection different pathways.
 * Where a property is different, the value is set to null.
 */
 seems like this is working 'as intended', but the ks4 subjects differ on so many PFs they're getting reset to null more often than not...has this just always been a bug? 👀  we really need a default rather than a null wherever this is being used to generate the share page?

Reply:
Yeah I'm thinking this could have been a previous bug, it seems to mainly be affected by tiers as ks4 English and History both have examboards. This function returns out the common props, the programmeSlug is passed to the share button which is used to navigate to the share page